RetailCoder.VBE/Refactorings/ExtractMethod/ExtractMethodModel.cs

```diff
@@ -141,19 +105,19 @@ private void Setup()
         
         public string SelectedCode { get; private set; }
 
-        private ObservableCollection<ExtractedParameter> _parameters;
-        public ObservableCollection<ExtractedParameter> Parameters
+        private ObservableCollection<ExtractMethodParameter> _parameters;
+        public ObservableCollection<ExtractMethodParameter> Parameters
         {
             get
             {
                 if (_parameters == null || !_parameters.Any())
                 {
-                    _parameters = new ObservableCollection<ExtractedParameter>();
+                    _parameters = new ObservableCollection<ExtractMethodParameter>();
                     foreach (var declaration in SourceVariables)
                     {
-                        _parameters.Add(new ExtractedParameter(declaration.AsTypeNameWithoutArrayDesignator,
-                            ExtractParameterNewType.PrivateLocalVariable,
-                            string.Concat(declaration.IdentifierName, declaration.IsArray ? "()" : string.Empty)));
+                        _parameters.Add(new ExtractMethodParameter(declaration.AsTypeNameWithoutArrayDesignator,
+                            ExtractMethodParameterType.PrivateLocalVariable,
```

Comment:
As mentioned in chat: I'd prefer to pass as parameter by default and allowing users to manually change to local variables. @rubberduck-vba/contributors opinions?